index.bs

```diff
@@ -269,6 +269,17 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
     attestation=], the [=credential key pair=] is also used as the [=attestation key pair=], see [=self attestation=]
     for details.
 
+: <dfn>Credential Private Key</dfn>
+:: The private key portion of an [=[RP]=]-specific <dfn>credential key pair</dfn>, generated by an [=authenticator=]. 
+    The exact location where the private key is stored may differ. In some cases, the private key may be always stored by
+    the [=authenticators=] and is never exported. A number of [=platform attached=] [=authenticators=] behaves in this manner. In 
+    other cases, the private key is never stored by the [=authenticator=] and is exported to the relying party as 
+    credential ID. Many [=cross-platform attached=] [=authenticators=] behave this way. In particular, all [=authenticators=] 
+    exercising the U2F protocol exports the private keys as credential ID. The public key portion of the
+    [=credential key pair=] is known as the <dfn>credential public key</dfn>. Note that in the case of [=self
+    attestation=], the [=credential key pair=] is also used as the [=attestation key pair=], see [=self attestation=]
+    for details.
+
```

Comment:
IMO this definition is both unnecessary and inaccurate.

As mentioned in my review of #378, I disagree with using keyStorage to specify the storage of the credential private key (i.e., which speaks to the authenticator's implementation) instead of its availability (which speaks to the authenticator's behavior).

Also, no authenticator exports the private key as the credential ID. Some may do a cryptographic wrap of the key material to produce a credential ID, but even that is an implementation choice that should not be enshrined in a specification.

Finally, the last two sentences just seem out of place here.